src/components/InputSection.jsx

```diff
@@ -22,8 +21,8 @@ function InputSection({
     <main>
       <section className="bg-[url('/src/img/input-section-bg.png')] bg-cover bg-center w-full h-screen border-y border-secondary-color">
         <Wrapper>
-          <div className="relative h-screen flex flex-col md:flex-row justify-center items-center">
-            <div className="md:grow text-sm md:text-base flex flex-col items-center space-y-1 text-center text-primary-dark">
+          <div className="md:grow text-sm md:text-base flex flex-row items-center space-y-1 text-center text-primary-dark mt-80">
+            <div className="md:grow text-sm md:text-base flex flex-col items-center space-y-1 text-center text-primary-dark w-1/2">
```

Comment:
We can adjust `w-1/2` to only apply this logic to large screens by prefixing it with `lg:` like so `lg:w-1/2`

This will improve part of the mobile issues we were facing after updating the previous comments code